### DISCUSSION TITLE
Conditional Rust tool-chain with Rust version

### DISCUSSION THREAD

**Discussion by kazuk** [sagiegurari/cargo-make#642] [Q&A] [answered]
I write coverage build make task.

https://github.com/kazuk/cargo-make-coverage-grcov

and, Rust 1.60 released. 

Rust 1.60 have stable support for profile instrumentation.

so I needs switch tool-chain with project rust version. if grater 1.60 then `Stable` else `Nightly`.

some one knows how to do?

**Comment by kazuk** [comment, marked as answer]
I found this example.

https://github.com/sagiegurari/cargo-make/blob/master/examples/toolchain.toml#L8

```
toolchain = { channel = "stable", min_version = "1.56" }
```

but this is failing build with toolchain condition not met.

**Reply by sagiegurari** [reply]
@kazuk sorry for really late reply. i usually monitor the issues. anyhow, did you find an answer to your requirement?

**Reply by kazuk** [reply]
Ready Fixed with this issue and related pr.
 https://github.com/sagiegurari/cargo-make/issues/643
